frontend_vue/src/components/ui/BannerCanarytools.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div class="@container">
+    <div
+      class="w-full gap-16 p-16 mt-16 rounded-xl banner-bg"
+      :class="[xlContainerClasses, mobileContainerClasses]"
+    >
+      <span class="hidden @sm:block"></span>
+      <p class="@sm:self-center self-start text-xl font-light text-white">
+        <span class="font-bold">Did you know</span> some of the best security
+        teams in the world run
+        <span class="font-bold">Thinkst Canary?</span>
+      </p>
+      <div class="flex self-end justify-end m-w-[0px]">
+        <BaseButton
+          class="whitespace-nowrap"
+          variant="text-light"
+          icon-position="right"
+          icon="arrow-right"
+          link="https://canary.tools/"
+          target="_blank"
+          >Find out</BaseButton
+        >
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts" setup>
+const xlContainerClasses =
+  'grid @sm:justify-center @sm:items-center @sm:grid-cols-[2fr_3fr_1fr] @sm:grid-rows-[2fr] @sm:aspect-[4/1] @md:h-[180px] @sm:bg-[position:-200px_-100px,0] @sm:bg-[size:550px,cover]';
+const mobileContainerClasses =
+  'grid-rows-[3fr_1fr] px-16 h-[280px] aspect-[1/2] bg-[position:-250px_0px,0] bg-[size:550px,cover]';
```

Comment:
we need to be careful doing this kind of thing... folks are going to reuse this elsewhere and we're going to drift back into spaghetti CSS... are we doing this kind of thing elsewhere?

Reply:
I'm not sure I understood the issue.
This banner can be placed everywhere and keeps the layout as it's shown in the preview of the diff. Ideally, nobody will touch this CSS
( I'm using container queries, not media queries )

Reply:
i don't want us to get into the habit of describing the CSS in variables that can be used in more than one place - else we may as well use standard CSS

Reply:
Oh I see, uhm, it just gets very hard to follow with tailwind ( It would be like 3 lines of classes )
but I can move them up into the template to avoid the possibility